### PR TITLE
Allow kubelogin to fetch wither access or id tokens.

### PR DESCRIPTION
**Pull request by ashish-amarnath**

### CODE REVIEW

File: cmd/server/main.go [hunk 1]
```diff
@@ -78,6 +79,14 @@ var (
 		[]string{"method"})
 )
 
+func getEnvOrDefault(envVar, defaultVal string) string {
+	val := defaultVal
+	if os.Getenv(envVar) != "" {
```
Comment (jkinkead) [inline cmd/server/main.go]: Alternately:

```go
if val, ok := os.LookupEnv(envVar); ok {
  return val
} else {
  return defaultVal
}
```

Comment (ashish-amarnath) [reply cmd/server/main.go]: Thank for the alternative. Will stick with the change for now. :)

File: Makefile [hunk 2]
```diff
@@ -8,7 +8,7 @@ GITHUB_REPO_HOST_AND_PATH := github.com/$(GITHUB_REPO_OWNER)/$(GITHUB_REPO_NAME)
 IMAGE_NAME := quay.io/nordstrom/kubelogin
 BUILD := build
 CURRENT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
-CURRENT_TAG := v0.0.55
+CURRENT_TAG := v0.0.6-dev
```
[the remark this answers is not in the document]
Comment (ashish-amarnath) [reply Makefile]: Updated. Thanks for catching that!

File: Dockerfile [hunk 3]
```diff
@@ -1,15 +1,13 @@
 FROM quay.io/nordstrom/baseimage-ubuntu:16.04
 
-ARG CURRENT_TAG=v0.0.55
+ARG CURRENT_TAG=v0.0.0
```
Comment (omadawn) [inline Dockerfile]: v0.0.0?

File: Makefile [hunk 4]
```diff
@@ -8,7 +8,7 @@ GITHUB_REPO_HOST_AND_PATH := github.com/$(GITHUB_REPO_OWNER)/$(GITHUB_REPO_NAME)
 IMAGE_NAME := quay.io/nordstrom/kubelogin
 BUILD := build
 CURRENT_BRANCH := $(shell git rev-parse --abbrev-ref HEAD)
-CURRENT_TAG := v0.0.55
+CURRENT_TAG := v0.0.6
```
Comment (omadawn) [inline Makefile]: From 0.0.55 to 0.0.6?
And it's 0.0.0 above?

File: Dockerfile (hunk 3 above)
Comment (ashish-amarnath) [reply Dockerfile]: That's the default value. An override for this value is supplied to the docker command using `build-arg` and this override is always expected.

File: Makefile (hunk 4 above)
Comment (ashish-amarnath) [reply Makefile]: This is a change that requires a change in the minor version. We would have gone to 56 if this were to be a minor edit like fixing a typo or an insignificant log message.

**Pull request merged**